card/src/main/res/values-cs-rCZ/strings.xml

```diff
@@ -32,6 +32,7 @@
 
     <string name="checkout_card_installments_title">Počet splátek</string>
     <string name="checkout_card_installments_option_regular">%s měsíců</string>
+    <string name="checkout_card_installments_option_regular_with_price">%s× %s</string>
```

Comment:
This looks like the Arabic `x`, should we verify if this is formatted correctly?

Reply:
I checked and it is formatted correctly